Sources/SmokeDynamoDB/DynamoDBCompositePrimaryKeyTable+bulkUpdateSupport.swift

```diff
@@ -284,4 +284,12 @@ extension DynamoDBCompositePrimaryKeyTable {
         let joinedElements = elements.joined(separator: ", ")
         return "{\(joinedElements)}"
     }
+
+    private func sanitizeString(_ string: String) -> String {
+        if self.escapeSingleQuoteInPartiQL {
+            return string.replacingOccurrences(of: "'", with: "''")
```

Comment:
Is this how the escaping works? Is there a link we can add that explains why we are doing this?

Reply:
Yes.

I cannot find any docs that say we need to do this. None of these talk about escaping quotes in string attributes.
- https://partiql.org/assets/PartiQL-Specification.pdf
- https://partiql.org/tutorial.html
- https://aws.amazon.com/blogs/opensource/announcing-partiql-one-query-language-for-all-your-data/
- https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/ql-reference.html

I can add a comment explaining why this is needed.

Reply:
I added a comment.

Reply:
Unfortunate there is no documentation to link to, thanks for adding the comment.